frontend/src/TopBar.js

```diff
@@ -0,0 +1,133 @@
+import React, {useState} from 'react';
+import {
+  AppBar,
+  Toolbar,
+  Typography,
+  Button,
+  IconButton,
+  MenuItem,
+  Drawer,
+  Divider
+} from '@material-ui/core';
+import {AccountCircle, Menu} from '@material-ui/icons'
+import {ThemeProvider,createMuiTheme} from '@material-ui/core/styles'
+import {green} from '@material-ui/core/colors'
+import {colors,TextField,InputAdornment,SvgIcon,makeStyles} from '@material-ui/core';
+import SearchIcon from '@material-ui/icons/Search';
+
+
+const theme = createMuiTheme({
+  palette:{
+    primary:{
+      main: green[500]
+    },
+    secondary: {
+      main: colors.lightBlue[500]
+    },
+    text: {
+      primary: colors.blueGrey[900],
+      secondary: colors.blueGrey[600]
+    }
+  },
+
+  typography :{
+    h1:{
+      fontFamily: "sans-serif",
+      fontSize:"1rem"
+    },
+    h3:{
+    fontFamily: "sans-serif",
+    fontSize:"1.3rem"
+    }
+  }
+
+  }
+)
+const TopBar = () => {
+  const [open,setOpen] = useState(false)
+  const toolbarStyle = {
+    minHeight: '65px',
+  };
+  const handleDrawer=()=>{
+    setOpen(true)
+  }
+  return (
+    <ThemeProvider theme = {theme}> 
+    <div>
+      <AppBar position='static' color='primary'>  
+        <Toolbar style={toolbarStyle}>
+            <IconButton onClick ={handleDrawer} color='inherit' edge='start' aria-label='Menu'>
+              <Menu>
+              </Menu>
+            </IconButton>
+            <Typography onClick={(e) => { e.preventDefault(); window.location.href='/';}} variant = 'h3' style={{flexGrow:1}}>
```

Comment:
Anytime that we use `window.location.href` it requests that page from the server which is not what we want. We can leave this for now but we'll go back and fix this later so that our Redux Store stays between page changes.